src/nanoemoji/color_glyph.py

```diff
@@ -328,8 +330,18 @@ def create(
                     font_config.ignore_reuse_error,
                 )
             )
+        if user_transform is None:
+            user_transform = Affine2D.identity()
+        print("user_transform", user_transform)
```

Comment:
all the print() will have to be replaced with equivalent logging.info() or debug()

Reply:
shouldn't be any when done, I just found them helpful to see when testing a couple of glyphs